components/bl-badge-component/component.json

```diff
@@ -0,0 +1,193 @@
+{
+  "id": "c_43f923fdd80e8ed727be59ea96811b64",
+  "name": "Badge",
+  "description": "Badge generates a small badge to the corner of its child(ren).",
+  "showInToolbox": true,
+  "faIcon": "certificate",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "name": "badgeLabel",
+      "label": "Badge Label",
+      "type": "text",
+      "handlerId": "badgeLabelLogic",
+      "handlerLabel": "Badge Label Logic",
+      "handlerDescription": "This is a handler for the logic to determine the label of the Badge. Value returned by the handler will be used as the label of the Badge.",
+      "dataBinding": true,
+      "showInSettings": true,
+      "hasLogicHandler": true
+    },
+    {
+      "type": "color",
+      "name": "badgeLabelColor",
+      "label": "Badge Label Color",
+      "showInSettings": true,
+      "hasLogicHandler": false
+    },
+    {
+      "type": "color",
+      "name": "badgeBackgroundColor",
+      "label": "Badge Background Color",
+      "showInSettings": true,
+      "hasLogicHandler": false
+    },
+    {
+      "name": "badgeFontSize",
+      "label": "Badge Font Size",
+      "type": "text",
+      "showInSettings": true,
+      "hasLogicHandler": false
+    },
+    {
+      "type": "select",
+      "name": "badgeAlignment",
```

Comment:
I think as we haven't any other Aligments, we could rename from `badgeAlignment` to `aligment`, at least in code(handler names)

Same for other properties